CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `"system_font"` feature gates reading system fonts. [#370]
+
+https://github.com/hecrj/iced/pull/370
```

Comment:
Thanks for updating the `CHANGELOG`! I believe we are missing a `[#370]:` prefix here.

Reply:
wow, hard to believe I missed that, fixing